docs/en/inference/vl_pipeline.md

```diff
@@ -114,6 +114,20 @@ print(response)
 
 For more information about customizing a chat template, please refer to [this](../advance/chat_template.md) guide
 
+### Setting vision model parameters
+
+The default parameters of the visual model can be modified by setting `VisonConfig`.
+
+```python
```

Comment:
example failed with
```
Traceback (most recent call last):
  File "wscript/test_vl_pipeline.py", line 5, in <module>
    pipe = pipeline('liuhaotian/llava-v1.5-7b', vision_config=vision_config)
  File "/projects/lmdeploy/lmdeploy/api.py", line 94, in pipeline
    return pipeline_class(model_path,
  File "/projects/lmdeploy/lmdeploy/serve/vl_async_engine.py", line 17, in __init__
    self.vl_encoder = ImageEncoder(model_path, vision_config)
  File "/projects/lmdeploy/lmdeploy/vl/engine.py", line 69, in __init__
    self.model = load_vl_model(model_path)
  File "/projects/lmdeploy/lmdeploy/vl/model/builder.py", line 36, in load_vl_model
    return LlavaVisionModel(model_path)
  File "/projects/lmdeploy/lmdeploy/vl/model/llava.py", line 61, in __init__
    self.build_model()
  File "/projects/lmdeploy/lmdeploy/vl/model/llava.py", line 96, in build_model
    load_checkpoint_and_dispatch(
  File "/opt/py38/lib/python3.8/site-packages/accelerate/big_modeling.py", line 490, in load_checkpoint_and_dispatch
    load_checkpoint_in_model(
  File "/opt/py38/lib/python3.8/site-packages/accelerate/utils/modeling.py", line 1294, in load_checkpoint_in_model
    raise ValueError(f"{param_name} doesn't have any device set.")
ValueError: model.embed_tokens.weight doesn't have any device set.
```

Reply:
the version of accelerate should be >= v0.29.3

Reply:
It works.

Reply:
@irexyc maybe we could check the version of accelerate in lmdeploy.vl and give the straight-forward message if the version is not met.